backend/app/testbot/chatbot_data/chatbot_A/products.md

```
@@ -0,0 +1,11 @@
+# E-commerce Products
+
+## Product: Widget A
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)